invenio_app_ils/circulation/views.py

```diff
@@ -157,7 +157,7 @@ def post(self, pid, record, **kwargs):
         is_overdue = days_ago > 0
         if not is_overdue:
             raise OverdueLoansMailError(description="This loan is not overdue")
-        send_loan_overdue_reminder_mail(record, days_ago)
+        send_loan_overdue_reminder_mail(record, days_ago, True)
```

Comment:
Can you please add the param name so that is more readable?
```suggestion
        send_loan_overdue_reminder_mail(record, days_ago, is_manually_triggered=True)
```